pagopa/io-app
# chore(IT Wallet): [SIW-603] Autoresizable info bottom sheet

## Short description
This PR proposes a change to the `useItwInfoBottomSheet.tsx` which now uses an autoresizable modal as base component. 

| Before | After |
|--------|--------|
| <img src="https://github.com/pagopa/io-app/assets/12371664/48d116c9-2878-402b-ad52-58dfba03366c" height="400"> | <img src="https://github.com/pagopa/io-app/assets/12371664/7b31b921-f9e0-4a8b-ac07-4ffaf63eabcd" height="400"> |



## List of changes proposed in this pull request
- Renames `useItwInfoFlow` to `useItwInfoBottomSheet`;
- Adds `deprecated` tag to `useItwDataProcessing.tsx` since it's a specific case of `useItwInfoBottomSheet`;
- Uses an autoresizable modal as base component for  `useItwInfoBottomSheet`;
- Adds a fixed `VSpacer` after the main `View` component of the bottom sheet;
- Updates usage accordingly.

## How to test
- Open the app; 
- Activate the wallet from the banner and obtain a PID; 
- Navigate to the wallet section and try to add any credential; 
- Press on the `i` button in the credential preview screen and the bottom sheet in the credential preview screen should show the whole content without scrolling; 
- Obtain the credential then go back to the wallet section; 
- Long press the credential and start the presentation flow; 
- Press on "Why" in the presentation data screen and the bottom sheet in the credential preview screen should show the whole content without scrolling.
